### ISSUE TITLE
Regardless of the variable the nicks feature changes the name at will. [Bug]

### ISSUE THREAD

**Issue by ChrisTVH** [Draycia/Chat#513]
## Bug Description:

The alias commands of my server are not working because of the nicknames, no matter if I put a variable that indicates a clean name the nickname comes out.

### What is not working as it should?

It always returns the nickname instead of a clean name.

### Steps to reproduce:

1. In my case activate the nicknames in carbon chat
2. Right now I am using CMI for alias commands.
3. Set up an alias command with any variable that returns a player name.

### System Details:

1. Server Type: Paper
2. Server Software:  git-Purpur-2175 (MC: 1.20.4)*
3. MC Version: 1.19.4 - 1.21
4. Carbon Version: 3.0.0-SNAPSHOT+3a93b78

### Pastebins:

![image](https://github.com/user-attachments/assets/b5b05183-7b42-48fb-a3d0-b208273a754b)
![image](https://github.com/user-attachments/assets/98643eda-aebe-43e6-a0ef-a9c39d94d2e3)

### Any other relevant details:

In short, no matter what variable I use it always returns the nickname and not the actual username.

**Comment by ChrisTVH** [comment]
Another thing I want to add, it only happens when I click on the npc if I execute the command alias it doesn't happen.

**Comment by ChrisTVH** [comment]
I reopen this issue, because I happen to notice, either the npcs plugin or any alias command that is run as a player no matter what variable I stress "no matter what variable is used" is always changed to the carbon nickname.

**Comment by jpenilla** [comment]
I am having a hard time deciphering what this issue is about. But I am not able to reproduce any issues with PlaceholderAPI.
Running `/papi parse me %player_name% %carbonchat_nickname_l%` returns 
![Screenshot 2024-07-20 at 9 42 50 AM](https://github.com/user-attachments/assets/6ee7c46b-6201-4ad2-8280-a01d168ef1a9), as expected. Maybe you don't have the Player PAPI addon?
Either way this issue cannot be looked into without a clear description of the problem and reproduction steps, with only PlaceholderAPI and Carbon installed (as mentioned in Discord).

**Comment by ChrisTVH** [comment]
Yes, I did that too and it came out correctly.

**Comment by ChrisTVH** [comment]
That is the same reason why I reopened it, because it is as if carbon changed the variable by the nickname instead of showing what the variable should give.

**Comment by ChrisTVH** [comment]
![image](https://github.com/user-attachments/assets/39fae341-f2b8-474d-8ff7-5fcb5ff2cf45)
Even if I use the variable %player_name% it is replaced by the nickname.
![image](https://github.com/user-attachments/assets/74c4d524-d4ef-4bb0-bb86-00a0ea103885)
![image](https://github.com/user-attachments/assets/5f0c6946-1343-45e5-9465-ef0ac2a683e3)
You can't just close the issue without trying to understand why I opened it in the first place.

**Comment by ChrisTVH** [comment]
And obviously I'm using placeholderapi, literally no plugin would work properly without that.
